### ISSUE TITLE
Request/Question: Can you render multiple pages at a time?

### ISSUE THREAD

**Issue by mansona** [ef4/prember#9]
I haven't looked into this in any great detail but I'm currently pre-rendering roughly 2K pages in an ember app (each version of the ember guides) and it takes a **long** time.

I wonder if it were possible to speed up the process to "batch request" 10 or 20 pages at a time?

**Comment by ef4** [comment]
Yes, this should be possible. Most likely we would want to fork multiple
worker processes.

**Comment by mansona** [comment]
I will take a look when (if?) this becomes an issue for us during deployment 👍

**Comment by midhunpm** [comment]
@ef4 - I am also trying to do the same thing. I am fairly new to Ember. Would you please explain what do you mean by "fork multiple worker processes"? Or any references to the explanation will also be helpful. 
Thank you!

**Comment by mansona** [comment]
@midhunpm how many pages are you looking to prerender? 

This isn't really an "Ember" thing per say 😂 it's more of a NodeJS thing. Essentially the original issue was related to "batching" the prerender process: https://github.com/ef4/prember/blob/master/lib/prerender.js#L122 to speed it up.

We eneded up not going down this line for the Ember Guides because we're rendering roughly 3K pages in about 10 mins every time we deploy https://travis-ci.org/ember-learn/guides-app/builds/461084204 and we figured that was "good enough" for our use cases 👍

**Comment by ef4** [comment]
Node has [experimental worker thread support](https://nodejs.org/dist/latest-v11.x/docs/api/worker_threads.html#worker_threads_worker_threads) now.

**Comment by midhunpm** [comment]
@mansona - I am trying to pre-render around 1400 pages. There is not much of dynamic content on these pages and it servers mostly like a brochure page to our application. And above all that to leverage SEO benefit we decided to pre-render these pages and serve them as static html files. I hope this is a good idea!

**Comment by mansona** [comment]
@midhunpm how often do you pre-render it? like I said we are getting a 10 mins build for double the number of pages and that suits our needs for now 👍

**Comment by midhunpm** [comment]
@mansona - The application is not production ready yet but we are thinking about running build every night to pre-render the pages as we might have changes in the content that we display in the pages.
